api_tests/nodes/filters/test_filters.py

```diff
@@ -94,3 +94,103 @@ def test_contributor_filter(self):
         res = self.app.get('{}{}'.format(self.contributors_url, self.user_two._id), auth=self.user.auth)
         actual = [node['id'] for node in res.json['data']]
         assert_equal(expected, actual)
+
+
+class NodesListDateFilteringMixin(object):
```

Comment:
Ah, to be as thorough as possible this should be subclassed on all views where nodes can be filtered (so user nodes and institution nodes as well). Looks like I forgot to do that though.